public/locales/en/settings.json

```diff
@@ -63,6 +63,17 @@
       "details": "Records JavaScript error messages and stack traces that occur while using the app, where possible. It is very helpful to know when the app is not working for you, but <1>error messages may include identifiable information</1> like CIDs or file paths, so only enable this if you are comfortable sharing that information with us."
     }
   },
+  "Experiments": {
+    "description": "Here you can get an early preview into new IPFS features by enabling options below. We are testing these ideas, and they may not be perfect yet so we'd love to hear your feedback.",
+    "issueUrl": "Open an issue",
+    "feedbackUrl": "💌Leave feedback",
+    "readMoreUrl": "Read More",
+    "npm": {
+      "title": "NPM-ON-IPFS",
+      "label": "Enable npm-on-ipfs",
+      "description": "Install npm modules through IPFS!"
```

Comment:
Perhaps something a bit more explanatory: `Install npm modules through IPFS with the 'ipfs-npm' command line tool`. I'm not good at writing this descriptions haha